test/unit/dom/SVGElement.js

```diff
@@ -0,0 +1,46 @@
+import canPatchNativeAccessors from '../../../src/util/can-patch-native-accessors';
+import create from '../../lib/create';
+
+describe('dom: SVGElement', function () {
+  runTests('div');
+  runTests('slot');
+  runTests('host');
+  runTests('root');
+
+  function runTests(type) {
+    describe(`${type}: `, () => {
+      let host, root, slot, div, elem;
+
+      beforeEach(function () {
+        host = document.createElement('div');
+        root = host.attachShadow({ mode: 'open' });
+        slot = create('slot');
```

Comment:
could just use native `document.createElement` here since no attrs/children are passed to `create(...)`